utils/calendrical_calculations/src/persian.rs

```diff
@@ -14,22 +14,17 @@ use crate::rata_die::RataDie;
 const FIXED_PERSIAN_EPOCH: RataDie = crate::julian::fixed_from_julian(622, 3, 19);
 
 /// Lisp code reference: <https://github.com/EdReingold/calendar-code2/blob/main/calendar.l#L4803>
-pub fn fixed_from_arithmetic_persian(year: i32, month: u8, day: u8) -> RataDie {
```

Comment:
nit: can we keep the old functions too?

Reply:
+1; note that since `calendrical_calculations` is a standalone crate, it can export functions even if we don't use them in ICU4X.

Reply:
Done. PTAL.